lms/urls.py

```diff
@@ -206,8 +206,11 @@
         continue
 
     # Make the assumptions that the templates are all in the same dir
-    # and that they all match the name of the key (plus extension)
-    template = "%s.html" % key.lower()
+    # and that they all match the name of the key
```

Comment:
I find this entire comment confusing -- how about rewording to something like "Append the default template extension to keys with unspecified extensions."